kbase-extension/static/kbase/js/widgets/function_output/kbaseSeqCompView.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Output widget for visualization of Sequence Comparison output
+ * @public
+ */
+
+define(['jquery',
+        'util/string',
+        'narrativeConfig',
+        'd3',
+        'kbwidget',
+        'kbaseAuthenticatedWidget',
+        'kbaseTabs',
+        'jquery-dataTables',
+        'jquery-dataTables-bootstrap'], function($, StringUtil, Config) {
+    $.KBWidget({
+        name: "kbaseSeqCompView",
+        parent: "kbaseAuthenticatedWidget",
+        version: "1.0.0",
+        ws_id: null,
+        ws_name: null,
+        token: null,
+        width: 1150,
+        options: {
+            ws_id: null,
+            ws_name: null
+        },
+        loadingImage: Config.get('loading_gif'),
+        wsUrl: Config.url('workspace'),
+        timer: null,
+
+        init: function(options) {
+            this._super(options);
+
+            this.ws_name = options.ws_name;
+            this.ws_id = options.ws_id;
+            if (options.report_name && options.workspace_name) {
+                  this.ws_id = options.report_name;
+                  this.ws_name = options.workspace_name;
+            }
+	    this.ws_id = 'ecoli.dnadiff.output';
+	    this.ws_name = 'fangfang:1454986986211';
```

Comment:
It looks like you're setting this.ws_id and this.ws_name here explicitly, which will override the user options...